pkg/machines/components/networks/createNetworkDialog.css

```diff
@@ -0,0 +1,11 @@
+.create-network-dialog-grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-row-gap: 1em;
+  grid-column-gap: 1em;
+  align: left;
```

Comment:
What's this?

Is it supposed to be `text-align: left;`, which is probably already the default? Or something else?

(As it isn't doing anything, you can remove this line.)